### PR TITLE
[web] Add DASD UI - first version

### PR DESCRIPTION
**Pull request by dgdavid**
Now that there is a D-Bus API for DASD (see https://github.com/yast/d-installer/pull/464), the web client can offer a UI for managing these devices.

Reviewers, please note that it is a very rough first version that lacks of unit testing. However, it has been manually tested and it works acceptable enough.

---

[Screencast from 2023-03-24 14-52-39.webm](https://user-images.githubusercontent.com/7056681/227561177-691eaed5-ce96-4f84-9791-37d5de0f6245.webm)

### CODE REVIEW

File: service/lib/dinstaller/dbus/storage/dasd.rb [hunk 1]
```diff
@@ -125,7 +160,9 @@ def dasd=(value)
           dbus_reader(:device_name, "s")
           dbus_reader(:formatted, "b")
           dbus_reader(:diag, "b")
+          dbus_reader(:status, "s")
```
Comment (ancorgs) [inline service/lib/dinstaller/dbus/storage/dasd.rb:163]: Please, update `doc/dbus_api.md`

File: web/src/assets/styles/patternfly-overrides.scss [hunk 2]
```diff
@@ -121,3 +121,12 @@ section .pf-c-toolbar {
   --pf-c-toolbar--PaddingBottom: 0;
   --pf-c-toolbar--PaddingTop: 0;
 }
+
+// FIXME: these main.stack PF4/toolbar overrides are just for testing.
```
Comment (ancorgs) [inline web/src/assets/styles/patternfly-overrides.scss:125]: Is this comment accurate?

Comment (dgdavid) [reply web/src/assets/styles/patternfly-overrides.scss:125]: No, it isn't. Probably a leftover :) :)

**Comment by teclator** [comment]
[Screencast from 2023-03-24 14-52-39.webm](https://user-images.githubusercontent.com/7056681/227561177-691eaed5-ce96-4f84-9791-37d5de0f6245.webm)

File: web/src/client/storage.js [hunk 3]
```diff
@@ -238,6 +245,257 @@ class ProposalManager {
...
+    // return this.assigned_client;
+    if (!this._client) {
+      this._client = new DBusClient(this.service, this.address);
+    }
+
+    return this._client;
+  }
+
+  async isSupported() {
+    const proxy = await this.managerProxy();
```
Comment (joseivanlopez) [inline web/src/client/storage.js:275]: For next iterations: we should use info from ObjectManager instead of trying to create the proxy.

**Review by joseivanlopez — APPROVED** [review]
LGTM

**Comment by imobachgs** [comment]
Congrats to everyone involved in this feature! 🎉

**Pull request merged**